src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -40,10 +42,232 @@ public void LanguageVersion()
             comp.VerifyDiagnostics();
         }
 
+        /// <summary>
+        /// System.IntPtr and System.UIntPtr definitions from metadata.
+        /// </summary>
+        [Fact]
+        public void TypeDefinitions_FromMetadata()
+        {
+            var source =
+@"interface I
+{
+    void F1(System.IntPtr x, nint y);
```

Comment:
Not related to this PR: have we decided whether `IntPtr` and `nint` should unify from a language perspective? 
The current implementation is that they don't. Is that intentional for LDM?

Reply:
Added PROTOTYPE comment to test type unification.

---
In reply to: [375001377](https://github.com/dotnet/roslyn/pull/40994#discussion_r375001377) [](ancestors = 375001377)